### DISCUSSION TITLE
distinguish between bainite and ferrite using mtex

### DISCUSSION THREAD

**Discussion by Mikewangkang** [mtex-toolbox/mtex#1915] [Ask Anything]
Hi all, I would like to know how to use mtex to differentiate bainite and ferrite in dual phase steels using kam values from ebsd results

**Comment by ralfHielscher** [comment]
Is there a accepted threshold value to discriminate between bainite and ferrite using the KAM?

Ralf.

**Reply by Mikewangkang** [reply]
yes

**Reply by Mikewangkang** [reply]
It is currently planned to adopt 5°.
